Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -316,6 +323,15 @@ void HTMLModelElement::sizeMayHaveChanged()
         createModelPlayer();
 }
 
+void HTMLModelElement::didUpdateLayerHostingContextIdentifier(ModelPlayer& modelPlayer, WebCore::LayerHostingContextIdentifier identifier)
+{
+    ASSERT_UNUSED(modelPlayer, &modelPlayer == m_modelPlayer);
+    ASSERT_UNUSED(identifier, identifier.toUInt64() > 0);
+
+    if (auto* renderer = this->renderer())
```

Comment:
@cdumez Per smart pointer rules, should this be a `CheckedPtr`?

Reply:
Is this going to be addressed?

Reply:
Yes! Will make it:
```
    if (CheckedPtr renderer = this->renderer())
        renderer->updateFromElement();
```